### PR TITLE
AbortController support

### PR DESCRIPTION
**Pull request by soukand**
Added AbortController support.
Seems that javascript world has finally agreed how to do cancellation.
https://developer.mozilla.org/en-US/docs/Web/API/AbortController

### CODE REVIEW

**Comment by nol13** [comment]
Awesome thanks! Like i mentioned I might tweak the docs a bit but looks good! Will give it another look in the morning but I think it'll work.

**Comment by nol13** [comment]
Also the types in fuzzball_lite.d.ts we updated but the changes aren't in fuzzball_lite.js yet.

(i know, is a bit of duplication)

File: README.md [hunk 1]
```diff
@@ -89,9 +89,23 @@ options.cancelToken = cancelToken;
...
         .catch((e) => {
-                if (e.message === 'canceled') console.log('I got canceled!') 
+                if (e.message === 'canceled') console.log('I got canceled!')
         });
 cancelToken.canceled = true;
+
+// or use AbortController to cancel search
+
+// Cancel search
+const abortController = new AbortController();
```
Comment (nol13) [inline README.md:99]: Hmm, maybe could just put a note here like..

// node 15+, non-IE browsers

**Comment by soukand** [comment]
@nol13 I added changes to fuzzball_lite aswell

**Pull request merged**